third_party/move/tools/move-package/src/compilation/compiled_package.rs

```diff
@@ -882,6 +903,7 @@ impl CompiledPackage {
             root_path: under_path.join(root_package.as_str()),
             package: OnDiskPackage {
                 compiled_package_info: self.compiled_package_info.clone(),
+                // TODO - bytecode deps
```

Comment:
For `OnDiskPackage`, I'm looking for suggestions. There are two main questions

- Whether to include bytecode deps in the same `dependencies` list or separately.
- Whether to save the `.mv` files of bytecode deps to disk, potentially alongside those compiled deps. This would break the current assumption that all on disk deps have sources and source maps.

@vgao1996